.github/workflows/publishing.yml

```diff
@@ -848,6 +848,12 @@ jobs:
       - name: Basic validation (GPU backends)
         shell: bash
         run: |
+          # Seems like we will need to install build-essential for resolving nbconvert issue `RuntimeError: Unsupported compiler -- at least C++11 support is needed!`"
+          # Given here: https://github.com/NVIDIA/cuda-quantum/blob/main/docs/sphinx/applications/python/qchem/cppe_lib.py#L18
+          # and https://github.com/jupyter/nbconvert/issues/1742
+          sudo apt-get update && sudo apt-get install build-essential python3-dev -y
+          python3 -m pip install cppe
```

Comment:
Q: do we need `pip install cppe` here since we also do that inside the notebook?